packages/material-ui/src/Popover/Popover.d.ts

```diff
@@ -28,7 +29,7 @@ export interface PopoverProps
   getContentAnchorEl?: null | ((element: HTMLElement) => HTMLElement);
   marginThreshold?: number;
   modal?: boolean;
-  ModalClasses?: ModalClassKey;
+  ModalClasses?: Partial<ClassNameMap<ModalClassKey>>;
```

Comment:
Whenever a component just passes props along I would prefer `PassComponentProps['passedProp']`. So in this case `ModalProps['classes']`. This makes it also clearer what this prop is used for and removes some noise due to importing utility types and `Partial`.

Reply:
I agree, we also need to fix https://github.com/mui-org/material-ui/blob/c5366234ab232c215f289754c948a4d1dee8cd3e/packages/material-ui/src/Menu/Menu.d.ts#L14 and https://github.com/mui-org/material-ui/blob/c5366234ab232c215f289754c948a4d1dee8cd3e/packages/material-ui/src/InputLabel/InputLabel.d.ts#L10